frontend/src/App.jsx

```diff
@@ -24,6 +24,9 @@ function App() {
   const [tgUser, setTgUser] = useState(JSON.parse(localStorage.getItem('tg_user')));
   const [showModal, setShowModal] = useState(false);
   const navigate = useNavigate();
+  const [isMobile, setIsMobile] = useState(false);
+  const [modal, setModal] = useState(true);
```

Comment:
rename it to be something more meaningful, like [isMobileRestrictionModalOpen, ..]

Reply:
renamed